zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -981,7 +980,7 @@ def get_postgresql_clusters(kube_client, cluster_id, alias, environment, region,
         else:
             ss = statefulset[0]
 
-        postgresql = [pg for pg in pgs if pg['name'] == version and pg['namespace'] == service_namespace]
+        postgresql = [pg for pg in postgresqls if pg['name'] == version and pg['namespace'] == service_namespace]
```

Comment:
what is the reason for this change?

Reply:
It used to be there for some workaround when the discovery functions were still generators (don't exactly remember why it was needed, but there was a reason).  Now that this is not the case anymore, I removed 'materializing' to a list.